scripts/test_translation_pipeline.py

```
@@ -0,0 +1,489 @@
+                unit_id=job.translation_units[0].unit_id,
+                agent_id="persist-agent",
+                translated_text="Prueba persistencia",
+                quality_score=0.85
+            )
+        
+        # Add to translation memory
+        manager1._add_to_translation_memory("Test", "Prueba", "en", "es", 0.8)
+        
+        # Add quality report
+        if job and job.translation_units:
+            manager1.add_quality_report(
```

Pylint (4.0.2)

[error] Instance of 'DistributedTranslationManager' has no 'add_quality_report' member

(E1101)